### PR TITLE
Remove universal support 4/n

### PR DESCRIPTION
**Pull request by fxcoudert**
Removing universal, PR 3/n: 27 changes out of a total of 147. This is split from #10337, so the testbot does not go crazy.

### CODE REVIEW

File: Formula/py2cairo.rb [hunk 1]
```diff
@@ -31,7 +29,7 @@ def install
     ENV.append_to_cflags `python-config --includes`
 
     # Python extensions default to universal but cairo may not be universal
```
Comment (ilovezfs) [inline Formula/py2cairo.rb]: This comment should be revised since universal is no longer a possibility here.

**Review by ilovezfs — APPROVED** [review]
assuming the py2cairo comment is updated

**Comment by fxcoudert** [comment]
@ilovezfs thanks for the review!

**Comment by ilovezfs** [comment]
You're welcome.

**Comment by ilovezfs** [comment]
Builds 1398 - 1424 on the Homebrew Testing job

**Pull request closed without merging**